docker-compose.debug.yml

```diff
@@ -36,8 +36,8 @@ services:
       - EZUNPAYWALL_NOTIFICATIONS_RECEIVERS
       - EZUNPAYWALL_NOTIFICATIONS_MACHINE=dev
       # unpaywall
-      - API_KEY_UPW=default
-      - UNPAYWALL_URL=http://fakeunpaywall:12000/feed/changefiles
+      - UNPAYWALL_APIKEY=default
+      - UNPAYWALL_HOST=http://fakeUnpaywall:12000
```

Comment:
Why changing from `URL` to `HOST` ? `http://fakeUnpaywall:12000` looks like a URL (where `fakeUnpaywall:12000` is the host)